src/liveShare/shareSession.ts

```diff
@@ -70,9 +70,9 @@ export function detachGuest(): void {
 }
 
 export function attachActiveGuest(): void {
-    if (config().get<boolean>('sessionWatcher')) {
+    if (config().get<boolean>('sessionWatcher', false)) {
```

Comment:
The default value of setting `r.sessionWatcher` is `true` already.

Reply:
Fixed in 5418748